test-e2e/tests/account/index_test.ts

```diff
@@ -1,35 +1,27 @@
-import passwordUtils, {LoginContext} from "../utils/password_utils";
```

Comment:
I don't know if it's going to be necessary to create directories for each test category. I'd probably prefer having a mixed hierarchy with the single files keeping their names and being in /tests and only having subfolders for categories that have multiple files.

Reply:
I agree, I fix this one